cel/env.go

```diff
@@ -193,12 +193,12 @@ func (e *Env) Compile(txt string) (*Ast, *Issues) {
 // Note, for parse-only uses of CEL use Parse.
 func (e *Env) CompileSource(src common.Source) (*Ast, *Issues) {
 	ast, iss := e.ParseSource(src)
-	if iss != nil && iss.Err() != nil {
+	if iss.Err() != nil {
 		return nil, iss
 	}
 	checked, iss2 := e.Check(ast)
-	if iss != nil && iss2 != nil {
-		iss.Append(iss2)
+	if iss2.Err() != nil {
+		iss = iss.Append(iss2)
 	} else if iss2 != nil {
```

Comment:
Probably this case is redundant. The issues list would be empty, so no `nil` checks should be needed

Reply:
Thanks, fixed.